coderedcms/static/coderedcms/scss/_crx-bs-overrides.scss

```diff
@@ -15,7 +15,7 @@
 
 // Carousel
 .container-fluid .carousel {
-  margin: 0 -15px;
+  margin: 0 -12px;
```

Comment:
Have you tried removing this rule entirely? This was originally created on bootstrap 4, so I'm not sure if it's still even needed.

Reply:
Yes I have - it is required and is not relevant to Bootstrap 4/5, it is a CRX override. Without the negative margin, the "full width" carousel has horizontal whitespace. The negative margin accounts for --bs-gutter-x in Bootstrap fluid containers to calculate left and right padding.

-15px of margin creates horizontal overflow.
-12px of margin removes overflow and makes carousel full width in fluid containers.
- Removing rule entirely creates horizontal whitespace, see attached.

![image](https://github.com/coderedcorp/coderedcms/assets/13271324/0aeea109-d542-489a-8f86-ba4ab0c00c25)